### ISSUE TITLE
3.5: aborts on checkout with "detected dubious ownership"

### ISSUE THREAD

**Issue by Reneator** [abarichello/godot-ci#93]
Using github actions i set up a workflow file that uses godot-ci.
When using the previous version 3.4.4 it works no problem (retested after this error) but with 3.5 it aborts:

![image](https://user-images.githubusercontent.com/24807557/183305753-5a364ff4-93dc-46e4-883a-178db76ecd85.png)

At the same location in the output for a build using godot and godot-ci 3.4.4:

![image](https://user-images.githubusercontent.com/24807557/183305705-0e528814-b1dc-4dcc-adf9-844a6773e9bb.png)

Here the workflow file im using for 3.5:

```
name: "Deploy Dev-Playtest Branch"
on:
  push:
    # Pattern matched against refs/tags
    tags:        
      - '*.dev*'

env:
  GODOT_VERSION: 3.5
  EXPORT_NAME: progress_game

jobs:
  export-windows:
    name: Windows Export
    runs-on: ubuntu-latest
    container:
      image: barichello/godot-ci:3.5
    steps:
      - name: Checkout
        uses: actions/checkout@v3.0.0
        with:
          lfs: true
      - name: Setup
        run: |
          mkdir -v -p ~/.local/share/godot/templates
          mv /root/.local/share/godot/templates/${GODOT_VERSION}.stable ~/.local/share/godot/templates/${GODOT_VERSION}.stable
      - name: Windows Build
        run: |
          mkdir -v -p build/windows
          godot -v --export "Windows Desktop" build/windows/${EXPORT_NAME}.exe
      - name: Upload Artifact
        uses: actions/upload-artifact@v3.0.0
        with:
          name: windows
          path: build/windows
 [...]
         
```
The file has some more extra steps, but it already aborts with the export-windows step.

When changing the version in the workflow file im changing both the godot_version and the godot-ci version

**Comment by Reneator** [comment]
The same is happening for 3.4.5:
![image](https://user-images.githubusercontent.com/24807557/183306033-07851634-80fb-4b02-970e-c2b5348aa9ed.png)

**Comment by Reneator** [comment]
Bumping the version of git checkout (actions/checkout) from 3.0.0 -> 3.0.2 fixed it for me to properly build on 3.4.5 and 3.5.

![image](https://user-images.githubusercontent.com/24807557/183306639-3378bc15-3457-41c8-898e-d3c4046706bc.png)

Maybe this can be helpful to someone else googling this problem^^
